configs/mot/ocsort/README.md

```diff
@@ -0,0 +1,30 @@
+# Observation-Centric SORT: Rethinking SORT for Robust Multi-Object Tracking
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+Multi-Object Tracking (MOT) has rapidly progressed with the development of object detection and re-identification. However, motion modeling, which facilitates object association by forecasting short-term trajec- tories with past observations, has been relatively under-explored in recent years. Current motion models in MOT typically assume that the object motion is linear in a small time window and needs continuous observations, so these methods are sensitive to occlusions and non-linear motion and require high frame-rate videos. In this work, we show that a simple motion model can obtain state-of-the-art tracking performance without other cues like appearance. We emphasize the role of “observation” when recovering tracks from being lost and reducing the error accumulated by linear motion models during the lost period. We thus name the proposed method as Observation-Centric SORT, OC-SORT for short. It remains simple, online, and real-time but improves robustness over occlusion and non-linear motion. It achieves 63.2 and 62.1 HOTA on MOT17 and MOT20, respectively, surpassing all published methods. It also sets new states of the art on KITTI Pedestrian Tracking and DanceTrack where the object motion is highly non-linear
+
+<!-- [IMAGE] -->
+
+## Citation
+
+<!-- [ALGORITHM] -->
+
+```latex
+@article{cao2022observation,
+  title={Observation-Centric SORT: Rethinking SORT for Robust Multi-Object Tracking},
+  author={Cao, Jinkun and Weng, Xinshuo and Khirodkar, Rawal and Pang, Jiangmiao and Kitani, Kris},
+  journal={arXiv preprint arXiv:2203.14360},
+  year={2022}
+}
+```
+
+## Results and models on MOT17
```

Comment:
Consider add more checkpoints.

NOTE!!: You didn't change the model link here. Contact Tao to upload your trained model to the openmmlab server.

Reply:
Maybe I can just reuse the YOLOX detector weights from ByteTrack?